cg/services/order_validation_service/workflows/rml/validation_rules.py

```diff
@@ -0,0 +1,25 @@
+from cg.services.order_validation_service.rules.sample.rules import (
+    validate_application_compatibility,
+    validate_application_exists,
+    validate_applications_not_archived,
+    validate_container_name_required,
+    validate_sample_names_available,
+    validate_sample_names_unique,
+    validate_volume_interval,
+    validate_volume_required,
+    validate_well_position_rml_format,
+    validate_well_positions_required,
+)
+
+SAMPLE_RULES: list[callable] = [
+    validate_application_compatibility,
+    validate_application_exists,
+    validate_applications_not_archived,
+    validate_container_name_required,
+    validate_sample_names_available,
+    validate_sample_names_unique,
+    validate_volume_interval,
+    validate_volume_required,
+    validate_well_position_rml_format,
+    validate_well_positions_required,
```

Comment:
Regarding this one, should we refactor the https://github.com/Clinical-Genomics/cg/blob/6e180f846cd136d6bd7ecc55b505c425e55bdfe5/cg/services/order_validation_service/rules/sample/utils.py#L26 to comply with the RML plate configuration? I would do that in another PR